styles/globals.scss

```diff
@@ -378,6 +383,14 @@ button:disabled {
   }
 }
 
+.nhsuk-summary-list > div > dd > a {
```

Comment:
Don't think I need this since swapping <a> for react-router-dom Link.
Will remove in next commit if so.